backend/main.py

```
@@ -1,37 +1,82 @@
+    email: str
+    password: str
+    
+class UserResponse(BaseModel):
+    email: str
+    role: str
+    name: str
+    
+# ---------- TEMP "DATABASE" ----------
+# For now: hardcoded users (later we connect real DB)
+FAKE_USERS = {
+    os.getenv("TEACHER_EMAIL"): {
```

Ruff (0.14.7)

Undefined name `os`

(F821)